builtin/providers/bitbucket/resource_default_reviewers.go

```diff
@@ -0,0 +1,114 @@
+package bitbucket
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+type Reviewer struct {
+	DisplayName string `json:"display_name,omitempty"`
+	UUID        string `json:"uuid,omitempty"`
+	Username    string `json:"username,omitempty"`
+	Type        string `json:"type,omitempty"`
+}
+
+type PaginatedReviewers struct {
+	Values []Reviewer `json:"values,omitempty"`
+}
+
+func resourceDefaultReviewers() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceDefaultReviewersCreate,
+		Read:   resourceDefaultReviewersRead,
+		Update: resourceDefaultReviewersUpdate,
+		Delete: resourceDefaultReviewersDelete,
+
+		Schema: map[string]*schema.Schema{
+			"username": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"repository": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"reviewers": {
+				Type:     schema.TypeSet,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+				Required: true,
```

Comment:
Is this actually Required? I don't see it being used in the Create - i can, however, see it being used in the Read. I therefore think this needs to be Computed: true